src/jit/lsra.cpp

```diff
@@ -4845,55 +4914,64 @@ void LinearScan::buildIntervals()
 #ifdef DEBUG
 void LinearScan::dumpVarRefPositions(const char* title)
 {
-    printf("\nVAR REFPOSITIONS %s\n", title);
-
-    for (unsigned i = 0; i < compiler->lvaCount; i++)
+    if (enregisterLocalVars())
```

Comment:
I generally avoid early returns where the additional indentation doesn't impair readability, but that's a matter of preference.